pom.xml

```diff
@@ -298,25 +290,6 @@
                 </executions>
```

Comment:
`build-helper-maven-plugin` configuration looks like it might be unnecessary now, or could be moved to parent POM.